libs/prebuilt/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -305,6 +318,13 @@ def create_react_agent(
             !!! Note
                 The graph will make a separate call to the LLM to generate the structured response after the agent loop is finished.
                 This is not the only strategy to get structured responses, see more options in [this guide](https://langchain-ai.github.io/langgraph/how-tos/react-agent-structured-output/).
+        pre_model_hook: An optional pre-model hook that will be called every time before the `agent` node (LLM-calling node) and will change the input to the LLM.
+            Useful for managing long message history (e.g., message trimming, summarization, etc.).
+            If provided, a special node will be added to the graph that runs before the `agent` node.
+            Pre-model hook must be a callable or a runnable that takes in current graph state and returns an update in the form of {"processed_messages": [...], **other_keys}.
```

Comment:
The `processed_messages` doesn't seem generic in comparison to the description so far

```suggestion
            Pre-model hook must be a callable or a runnable that takes in current graph state and returns an update in the form of {"processed_messages": [...], **other_keys}.
```